functions/prepared-check-sync/v1.js

```diff
@@ -10,7 +10,7 @@ const preparedCheckSchemaValidator = require('../lib/prepared-check-schema-valid
 
 const v1 = {}
 
-v1.process = async function process (context, preparedCheckSyncMessage) {
```

Comment:
I would prefer a named function over an anonymous one.  It makes more sense when profiling the app.